docs/references/supported-signatures.mdx

```diff
@@ -13,17 +13,18 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
 ICP currently supports the following cryptographic signature types:
 
-- [ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa): ECDSA is supported via the ICP's novel threshold ECDSA protocol. Threshold ECDSA uses private ECDSA keys that only exist as secret shares that are distributed across the nodes within the threshold ECDSA subnet. Signatures are generated by leveraging these secret shares without reconstructing the subnet secret key. [Learn more about t-ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa)
+- [ECDSA](/docs/current/developer-docs/smart-contracts/signatures/t-ecdsa): ECDSA is supported via the ICP's novel threshold ECDSA protocol. Threshold ECDSA uses private ECDSA keys that only exist as secret shares that are distributed across the nodes within the threshold ECDSA subnet. Signatures are generated by leveraging these secret shares without reconstructing the subnet secret key. [Learn more about t-ECDSA](/docs/current/developer-docs/smart-contracts/signatures/t-ecdsa)
 
 - Boneh-Lynn-Shacham (BLS): ICP's chain-key cryptography is based on BLS signatures. BLS signatures are used for [canister signatures](/docs/current/concepts/glossary/#canister-signature) and [certified variables](/docs/current/concepts/glossary/#certified-variable). While BLS signatures have distinct benefits and advantages, they cannot be used with other blockchains. To integrate with other chains, ICP uses threshold signatures like threshold ECDSA.
 
-## Future additions
+- [Schnorr](/docs/current/developer-docs/smart-contracts/signatures/threshold-schnorr)
```

Comment:
I'm not sure what "Schnorr" in the link text refers to. BIP340?

Reply:
This is a typo, I meant to remove this line.